### PR TITLE
Python Source Code Optimization

### PR DESCRIPTION
**Pull request by AxylumRust**

### CODE REVIEW

File: utils/bypass.py [hunk 1]
```diff
@@ -101,21 +90,16 @@ def table_processing(statuses, wb_result, details, replay):
...
+            err = f'An incorrect response was received while processing payload {plp} in {zone}: {status_code}'
+            print(f'{Fore.YELLOW}{err}{Style.RESET_ALL}')
+    elif blocked:
+        status = ['PASSED', status_code] if status_code in block_code else ['BYPASSED', status_code]
     else:
-        if blocked:
-            status = ['PASSED', status_code] if status_code in block_code else ['BYPASSED', status_code]
-        else:
-            status = ['PASSED', status_code] if status_code not in block_code else ['FALSED', status_code]
+        status = ['PASSED', status_code] if status_code not in block_code else ['FALSED', status_code]
```
Comment (Romanov-R) [inline utils/bypass.py:102]: BYPASSED losed

**Review by Romanov-R — COMMENTED** [review]
Thanks, but It looks like code formatting, not code optimization
The commit contains errors

**Comment by AxylumRust** [comment]
Hello,
There is some code formatting and also some optimization.
Could you please tell me where the error is?

**Comment by Romanov-R** [comment]
> Hello, There is some code formatting and also some optimization. Could you please tell me where the error is?

I don't find any optimization, could you provide an example?

Error example:
https://github.com/nemesida-waf/waf-bypass/pull/19#pullrequestreview-1728999911

**Pull request closed without merging**